README.md

```diff
@@ -15,6 +15,8 @@ This image will create a zip archive of the project, send it to a `CODEBUILD_S3_
 
 Bitbucket|GitLab pipelines will wait for AWS CodeBuild to finish and return success or failure based on the outcome of the build.
 
+This image will download the zipped version of the codebuild artifacts and unzip it to .codebuild_artifacts 
```

Comment:
Oups this part is not good anymore : 

This image will download the codebuild artifacts and unzip it to .codebuild_artifacts

Reply:
solved in #2